HaulageApplication/Haulage/BaseClasses/Accounting/Administrator.cs

```diff
@@ -35,21 +35,82 @@ public void manageManifests()
 
         }
 
-        public void planRoutes()
+        public void AddManifestItem(TripManifest manifest, ManifestItem item)
         {
+            manifest.Items.Add(item);
+        }
 
+        public void RemoveManifestItem(TripManifest manifest, ManifestItem item)
+        {
+            manifest.Items.Remove(item);
         }
 
-        public void resourceTrips()
+        public void UpdateManifestItem(TripManifest manifest, ManifestItem oldItem, ManifestItem newItem)
```

Comment:
would it be helpful if this method returned some status about whether the update succeeded or not?

Reply:
LJ - Updated now.